engineering/conventions-go.md

```diff
@@ -62,4 +62,10 @@ A blank line is usually added after the end of any control structure, too.
 
 ### Naming
 
-* Prefer full names (e.g. `Repository`, not `Repo`).
\ No newline at end of file
+* Prefer full names (e.g. `Repository`, not `Repo`).
+
+## Configuration
+
+* Use environment variables for configuration.
+* You can use the [configurable](https://github.com/src-d/framework/tree/master/configurable) package  ([example](https://github.com/src-d/framework/blob/master/database/database.go)).
```

Comment:
I don't know if propagate `framework` to new projects is a good, idea.

Reply:
Sure. `configurable` package is so small we can probably go ahead and recommend [`envconfig`](https://github.com/kelseyhightower/envconfig)?